voice/voice-ai/x/AppConfigration/AppConfig.swift

```diff
@@ -46,6 +64,14 @@ class AppConfig {
         return self.deepgramKey
     }
     
+    // Getter methods for the review prompt configuration
+    func getMinimumSignificantEvents() -> Int? {
```

Comment:
could just return Int since default value is present

Reply:
actually we should not set the defalut value. it should be private var minimumSignificantEvents: Int?
    private var daysBetweenPrompts: Int?.